src/kalman/kalmanFilter.cpp

```diff
@@ -59,9 +59,7 @@ roboteam_msgs::World kalmanFilter::getWorld() {
             world.them.push_back(kalmanTheirBot.as_message());
         }
     }
-    if (ball.getExistence()){
-        world.ball=ball.as_ball_message();
-    }
+    world.ball=ball.as_ball_message();
```

Comment:
why remove the if?

Reply:
Because this is what we agreed with AI; the world.ball field is not empty regardless by design and otherwise we will send the ball at {0.0,0.0} as these are the default values. Now we simply send the last known position, which is better.